src/main/java/seedu/duke/data/timetable/WeeklyEvent.java

```diff
@@ -3,15 +3,18 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 
+/**
+ * Subclass of RecurringEvent that re-occurs every week.
+ */
 public class WeeklyEvent extends RecurringEvent {
     private static final int TIME_STEP = 1;
 
-    public WeeklyEvent(String title, LocalDateTime dateTime, boolean isToRemind, LocalDate endRecurrance) {
-        super(title, dateTime, isToRemind, endRecurrance, RecurringEvent.WEEKLY_RECURRENCE);
+    public WeeklyEvent(String title, LocalDateTime dateTime, boolean isToRemind, LocalDate endRecurrence) {
+        super(title, dateTime, isToRemind, endRecurrence, RecurringEvent.WEEKLY_RECURRENCE_TYPE);
```

Comment:
endRecurrence should be all caps right if this is for the final? Think got a few files to amend in that case

Reply:
Yes but in this case these are parameters not the constant so I don't think there's any coding standard violations.